gym/envs/classic_control/utils.py

```diff
@@ -0,0 +1,46 @@
+"""
+Utility functions used for classic control environments.
+"""
+
+from typing import Optional, Union
+
+import numpy as np
+
+
+def verify_number_and_cast(x: Union[int, float, np.ndarray]) -> float:
```

Comment:
One final comment - change the type annotation to `x: SupportsFloat` (it's from `typing` and is used in a few other places in gym). It should actually perform the same validation at "compile time" and from the IDE

Reply:
done. didn't know about this type, but useful to know for the future, thanks!